### PR TITLE
Fixed remaining lint errors. Now lints clean.

### PR DESCRIPTION
**Pull request by jordanh**

### CODE REVIEW

File: src/client/Root.js [hunk 1]
```diff
@@ -9,27 +9,25 @@ import {ensureState} from 'redux-optimistic-ui';
 const lookConfig = Presets['react-dom'];
 lookConfig.styleElementId = '_look';
 
-process.env.NODE_ENV !== 'production' &&
+if (!__PRODUCTION__) {
   lookConfig.plugins.push(Plugins.friendlyClassName);
```
Comment (jordanh) [inline src/client/Root.js]: By the way, I haven't seen this work in development mode. Not sure why. Not super worried about it, but it isn't interesting...

File: src/server/Html.js [hunk 2]
```diff
@@ -1,68 +1,58 @@
...
-        backgroundColor: 'red',
-        'specialProp=true': {
-            fontSize: 30
-        },
-        // On iOS the font is 'Helvetica Neue'
-        '@platform ios': {
-            fontFamily: 'Helvetica Neue'
-        }
-    }
-});
```
Comment (jordanh) [inline src/server/Html.js]: Glad I caught cleaning this up :)

File: src/server/Html.js [hunk 3]
```diff
@@ -1,68 +1,58 @@
...
 
-    const root = PROD && renderToString(
-      <LookRoot config={lookConfig}>
-        <Provider store={store}>
-          <RouterContext {...renderProps}/>
-        </Provider>
-      </LookRoot>
-    );
-
-    const otherCSS = StyleSheet.renderToString(lookConfig.prefixer);
```
Comment (jordanh) [inline src/server/Html.js]: This line also wasn't needed.

File: src/server/routesOrPrerender.js [hunk 4]
```diff
@@ -26,6 +31,6 @@ export default option => {
       />
     );
     const appCSS = StyleSheet.renderToString(lookConfig.prefixer);
-    res.send('<!DOCTYPE html>' + htmlString.replace(lookCSSToken, appCSS));
+    res.send(`<!DOCTYPE html>${htmlString.replace(lookCSSToken, appCSS)}`);
```
Comment (jordanh) [inline src/server/routesOrPrerender.js]: Any objection to using template strings here @mattkrick?

File: src/server/Html.js [hunk 5]
```diff
@@ -1,68 +1,58 @@
 /* eslint react/no-danger:0 */
-import React, {Component, PropTypes} from 'react';
-import { LookRoot, StyleSheet } from 'react-look';
+import React, {PropTypes} from 'react';
+import { LookRoot } from 'react-look';
```
Comment (mattkrick) [inline src/server/Html.js]: the spacing around LookRoot didn't throw a lint error?

Comment (jordanh) [inline src/server/Html.js]: No, I think @jrwells may have stricken that rule. Either way, I'm ok with it.

**Pull request merged**